noir_stdlib/src/embedded_curve_ops.nr

```diff
@@ -75,6 +86,9 @@ pub fn multi_scalar_mul<let N: u32>(
 // docs:end:multi_scalar_mul
 {}
 
+#[foreign(multi_scalar_mul)]
+pub fn multi_scalar_mul_slice(points: [EmbeddedCurvePoint], scalars: [EmbeddedCurveScalar]) -> [Field; 3] {}
+
```

Comment:
```suggestion
#[foreign(multi_scalar_mul)]
fn multi_scalar_mul_slice(points: [EmbeddedCurvePoint], scalars: [EmbeddedCurveScalar]) -> [Field; 3] {}

```

Happy to have this internally as a hack but I'd like to avoid having to expose two different flavours of this to the outside world.

Reply:
I put it as pub(crate), else it gives a warning about using a private function. I can also move it to hash/mod.nr if you prefer.